### DISCUSSION TITLE
Custom Domain: outstanding validation records

### DISCUSSION THREAD

**Discussion by jdamon96** [supabase/supabase#20806] [Questions]
I'm having an issue setting up the second of the two TXT validation records to implement a custom domain for my Supabase project. 

I'm following the instructions here: https://supabase.com/docs/guides/platform/custom-domains (replacing the project ref id & example.com values w/ my own)

```
supabase domains create --project-ref abcdefghijklmnopqrst --custom-hostname api.example.com
```

As the documentation says, two TXT records get returned:

```
Required outstanding validation records:
        _cf-custom-hostname.api.example.com TXT -> 46BBC14D-D50A-409C-8DB5-F862CF5BA660
        api.example.com TXT -> ca3-F1HvR9i938OgVwpCFwi1jTsbhe1hvT0Ic3efPY3Q
```

After adding the above records (again, replacing example.com with my specific domain), the first TXT record (_cf-custom-homename) is validated, but the second continues to be outstanding:

```
supabase domains reverify --project-ref <my_project_id>
Custom hostname verification in-progress; please configure the appropriate DNS entries and request re-verification.
Required outstanding validation records: 
	<custom_generated_supabase_string>.api.example.com TXT -> <record_supabase_generated_record_value> (replace any existing CNAME records)
```

If I check on https://mxtoolbox.com/SuperTool.aspx I see the DNS record and confirmed it resolves to the string value provided by Supabase

Has anyone run into this before? Any tips on how to resolve?

**Comment by DarkNeedle** [comment]
It's a real issue. I have been having the same issue and opened up a support ticket. After two days still no definitive response. This is an issue coming back from a Lets Encrypt DNS validation for the cert to be issued.

**Comment by rookledookle** [comment]
Same issue here

**Comment by codistcreative** [comment]
I had this same issue ongoing (using AWS R53 DNS) The only thing that worked was dropping the TTL from the default 300 to 60 for the TXT _acme_challenge record.

**Comment by mdruuu** [comment]
Anyone have a straight solution for this? Running into verification issue. as well. Given that they charge extra $10 a month for this, it's not great that setting it up is such a PITA.

**Reply by avitorio** [reply]
Did you ever find a solution?

**Comment by Playful-AI-Rich** [comment]
Bump.  Experiencing this now.
In current implementation, it doesn't even show you the first record, i had to run CLI in debug raw output mode to see the first validation record.  nothing i'm doing (including restarting hte process, and changing to different subdomains) is working.  have message out to support but apparently this is still a problem.

Anybody find a repeatable solution to this?

**Comment by interstates21** [comment]
Same issue

**Comment by dylanjmcdonald** [comment]
Experiencing this issue at the moment as well. whatsmydns shows the record has propagated but Supabase verification is not working.
